src/components/LifeEventsListView.vue

```diff
@@ -78,10 +100,24 @@
                  class="mt-2"
              />
            </v-list-item>
+           <v-list-item>
+            <div class="d-flex">
+              <v-checkbox-btn
+                v-model="newTodo"
```

Comment:
Das Data-Feld `newTodo` wird wohl nicht benötigt. Die Checkbox in der UI ist auch ohne Funktion. 

Ich schlage vor die Checkbox hier komplett zu entfernen und das `v-text-field` mit `.ml-?` entsprechend einrücken.

Reply:
Hab das Datafield entfernt, stattdessen die Checkbox disabled gemacht. Irgendwie passt für mich die Zeile mit Checkbox besser in das Layout...